mappings/net/minecraft/client/model/Model.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_3879 net/minecraft/client/model/Model
 	FIELD field_21343 layerFactory Ljava/util/function/Function;
 	METHOD <init> (Ljava/util/function/Function;)V
 		ARG 1 layerFactory
+	METHOD accept addPart (Ljava/lang/Object;)V
```

Comment:
```suggestion
```
Unfortunately this method overrides java consumer, so you cannot rename it.

Reply:
welp. Thanks, Mojang.